ci/setup-mingw-x86.ps1

```diff
@@ -0,0 +1,20 @@
+Set-StrictMode -Version Latest
+
+$ErrorActionPreference = "Stop"
+$PSDefaultParameterValues['*:ErrorAction'] = 'Stop'
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+[Reflection.Assembly]::LoadWithPartialName("System.IO.Compression.FileSystem");
+
+Write-Host "##############################################################################"
+Write-Host "## Downloading mingw"
+Write-Host "##############################################################################"
+
+$mingw_uri = "https://bintray.com/libgit2/build-dependencies/download_file?file_path=mingw-w64-i686-8.1.0-release-win32-sjlj-rt_v6-rev0.zip"
+$platform = "x86"
```

Comment:
Both scripts are essentially the same, except for "s/x86/x86_64/" and "s/i686/x86_64", right? If so we could just have it accept a parameter "arch" instead of duplicating it

Reply:
And the filename, which I'd rather embed in the script than in the yaml.

Reply:
Which can still easily be done, right?

```
case "$ARCH in
    x86)
        mingw_uri="https://..."
        platform=x86
        ;;
    amd64)
        ...
esac
```

(Sorry for using Bash, I'm not familiar with PS)

Reply:
Yes, I agree - this is a good suggestion.